Cubical/Data/Int/Properties.agda

```diff
@@ -300,6 +316,102 @@ private
+neg+ (suc m) zero = neg (suc (m +ℕ zero)) ≡⟨ cong neg (cong suc (+-zero m)) ⟩
+                       neg (suc m) + neg zero ∎
+neg+ (suc m) (suc n) = neg (suc m +ℕ suc n) ≡⟨ negsuc+ m (suc n) ⟩
+                       neg (suc m) + neg (suc n) ∎
+
+ℕ-anticomm : ∀ m n → m ℕ- n ≡ - (n ℕ- m)
+ℕ-anticomm zero zero = refl
+ℕ-anticomm zero (suc n) = refl
+ℕ-anticomm (suc m) zero = refl
+ℕ-anticomm (suc m) (suc n) = suc m ℕ- suc n ≡⟨ ℕ-anticomm m n ⟩
+                             - (suc n ℕ- suc m) ∎
+
+pos- : ∀ m n → m ℕ- n ≡ pos m - pos n
+pos- zero zero = refl
+pos- zero (suc n) = zero ℕ- suc n ≡⟨ +-comm (negsuc n) (pos zero) ⟩
+                    pos zero - pos (suc n) ∎
+pos- (suc m) zero = refl
+pos- (suc m) (suc n) = suc m ℕ- suc n                       ≡⟨ pos- m n ⟩
+                       pos m - pos n                        ≡⟨ sym (sucPred (pos m - pos n)) ⟩
+                       sucInt (predInt (pos m - pos n))     ≡⟨ cong sucInt (+predInt (pos m) (- pos n)) ⟩
+                       sucInt (pos m + predInt (- (pos n))) ≡⟨ cong sucInt (cong (pos m +_) (predInt-pos n)) ⟩
+                       sucInt (pos m + negsuc n)            ≡⟨ sucInt+negsuc n (pos m) ⟩
+                       pos (suc m) - pos (suc n) ∎
+
+-anticomm : ∀ m n → m - n ≡ - (n - m)
+-anticomm (pos n) (pos n₁) = pos n - pos n₁ ≡⟨ sym (pos- n n₁) ⟩
+                             n ℕ- n₁        ≡⟨ ℕ-anticomm n n₁ ⟩
+                          - (n₁ ℕ- n)       ≡⟨ cong -_ (pos- n₁ n) ⟩
+                          - (pos n₁ - pos n) ∎
+-anticomm (pos n) (negsuc n₁) = pos n - negsuc n₁     ≡⟨ +-comm (pos n) (pos (suc n₁)) ⟩
+                                pos (suc n₁) + pos n  ≡⟨ sym (pos+ (suc n₁) n) ⟩
+                                pos (suc n₁ +ℕ n)     ≡⟨ sym (-neg (suc n₁ +ℕ n)) ⟩
+                             -  neg (suc n₁ +ℕ n)     ≡⟨ cong -_ (neg+ (suc n₁) n) ⟩
+                             - (neg (suc n₁) + neg n) ≡⟨ cong -_ (cong (negsuc n₁ +_) (sym (-pos n))) ⟩
+                             - (negsuc n₁ - pos n) ∎
+-anticomm (negsuc n) (pos n₁) = negsuc n - pos n₁     ≡⟨ sym (negsuc+ n n₁) ⟩
+                                negsuc (n +ℕ n₁)      ≡⟨ cong -_ (pos+ (suc n) n₁) ⟩
+                             - (pos (suc n) + pos n₁) ≡⟨ cong -_ (+-comm (pos (suc n)) (pos n₁)) ⟩
+                             - (pos n₁ - negsuc n) ∎
+-anticomm (negsuc n) (negsuc n₁) = negsuc n - negsuc n₁        ≡⟨ +-comm (negsuc n) (pos (suc n₁)) ⟩
+                                   pos (suc n₁) + negsuc n     ≡⟨ sym (pos- (suc n₁) (suc n)) ⟩
+                                   suc n₁ ℕ- suc n             ≡⟨ ℕ-anticomm (suc n₁) (suc n) ⟩
+                                - (suc n ℕ- suc n₁)            ≡⟨ cong -_ (pos- (suc n) (suc n₁)) ⟩
+                                - (pos (suc n) - pos (suc n₁)) ≡⟨ cong -_ (+-comm (pos (suc n)) (negsuc n₁)) ⟩
+                                - (negsuc n₁ - negsuc n) ∎
+
+-dist : ∀ m n → - (m + n) ≡ (- m) + (- n)
```

Comment:
Maybe `-Dist`?

Reply:
Actually the name should also include `+` as there are two similar lemmas for multiplication

Reply:
Agreed. Went with `-Dist+`.